chain/vm/runtime.go

```diff
@@ -60,38 +61,55 @@ type Runtime struct {
 }
 
 func (rt *Runtime) TotalFilCircSupply() abi.TokenAmount {
-	total := types.FromFil(build.TotalFilecoin)
+
+	filVested, err := rt.vm.GetVestedFunds(rt.ctx)
+	if err != nil {
+		rt.Abortf(exitcode.ErrIllegalState, "failed to get vested funds for computing total supply: %s", err)
+	}
 
 	rew, err := rt.state.GetActor(builtin.RewardActorAddr)
 	if err != nil {
 		rt.Abortf(exitcode.ErrIllegalState, "failed to get reward actor for computing total supply: %s", err)
 	}
 
+	filMined := types.BigSub(types.FromFil(build.TotalFilecoin), rew.Balance)
```

Comment:
I don't think we should use build.TotalFilecoin here, would love to standardize the naming too. Can we rename TotalFilecoin to something like `FIL_BASE` and we should be using `FIL_Alloc_StorageMining: 0.55 * FIL_BASE` here. 

@ZenGround0 let's update actors when this PR lands

Reply:
@zixuanzh I agree that saying `filMined = FIL_BASE - rew.Balance` is a lit, but I'm uncertain about what the right think to do is (I think it'll depend on whether the Burnt Funds Actor always necessarily has 0 balance at genesis). 

Let's discuss it "in-person" tomorrow?

Reply:
@ZX is the current state of actors good wrt this change?  Is there some action required still?